packages/clay-card/src/CardWithNavigation.tsx

```diff
@@ -49,13 +54,18 @@ interface IProps {
 	title?: React.ReactText;
 }
 
+const KEYCODES = {
+	ENTER: 13,
+	SPACE: 32,
+};
+
 export const ClayCardWithNavigation: React.FunctionComponent<IProps> = ({
 	children,
 	description,
 	horizontal = false,
 	horizontalSymbol = '',
 	href,
-	onClick,
+	onClick: onClickCallback,
```

Comment:
Why not just keep `onClick`?

Reply:
I forgot to remove this alias, I was using in a older implementation 😅

Reply:
haha np